### ISSUE TITLE
Using ATSS to train the VOC data set loss has always been NAN

### ISSUE THREAD

**Issue by scuizhibin** [open-mmlab/mmdetection#6828]
atss_r50_fpn_1x_coco.py
_base_ = [
    '../_base_/datasets/voc0712.py',
    '../_base_/schedules/schedule_1x.py', '../_base_/default_runtime.py'
]

The loss is ;
lr: 1.000e-02, eta: 17:28:39, time: 0.231, data_time: 0.004, memory: 2024, loss_cls: 21.8341, loss_bbox: nan, loss_centerness: nan, loss: nan

**Comment by ZwwWayne** [comment]
ATSS might be unstable with small dataset. You may consider use it with an ATSS detector pre-trained on COCO.
